lnrpc/rpc.proto

```diff
@@ -1153,13 +1153,6 @@ message GetInfoResponse {
     uint32 num_inactive_channels = 15 [json_name = "num_inactive_channels"];
 }
 
-message ConfirmationUpdate {
```

Comment:
The message currently isn't used, but recently we've gained the ability to actually populate it. Would prefer to leave it in tact (but possibly remove the entry in the oneof so the docs are accurate).

Reply:
Reverted delete.